src/core/dataframe.js

```diff
@@ -61,26 +61,45 @@ export default class Dataframe {
         gl.bufferData(gl.ARRAY_BUFFER, ids, gl.STATIC_DRAW);
     }
 
-    setStyle(style) {
-        this.style = style;
-    }
-
-    getFeaturesAtPosition(pos) {
+    getFeaturesAtPosition(pos, style) {
         switch (this.type) {
             case 'point':
-                return this._getPointsAtPosition(pos);
+                return this._getPointsAtPosition(pos, style);
             case 'line':
-                return this._getLinesAtPosition(pos);
+                return this._getLinesAtPosition(pos, style);
             case 'polygon':
-                return this._getPolygonAtPosition(pos);
+                return this._getPolygonAtPosition(pos, style);
             default:
                 return [];
         }
     }
 
-    _getPointsAtPosition(pos) {
-        console.log(pos);
-        return [];
+    _getPointsAtPosition(p, style) {
+        p = wToR(p.x, p.y, { center: this.center, scale: this.scale });
```

Comment:
`strokeWidth` is not taken into account here. I think we should add it to cover the whole point.

Reply:
Ups, good catch, I totally forgot about that.

Reply:
Solved!